core/src/data/dataSource.h

```diff
@@ -105,6 +115,12 @@ class DataSource : public std::enable_shared_from_this<DataSource> {
     // Name used to identify this source in the style sheet
     std::string m_name;
 
+    // Minimum zoom for which tiles will be displayed
+    int32_t m_minDisplayZoom;
```

Comment:
Ohh I forgot about these. You can leave it 32 bit right now. Though I think all these can be 8 bit but I might have to look as why we had it as 32 in the beginning.